packages/fiori/src/SideNavigation.ts

```diff
@@ -16,101 +25,38 @@ import {
+import "@ui5/webcomponents-icons/dist/history.js";
+import "@ui5/webcomponents-icons/dist/locate-me.js";
+import "@ui5/webcomponents-icons/dist/calendar.js";
+import "@ui5/webcomponents-icons/dist/chain-link.js";
+
 // Styles
-import SideNavigationCss from "./generated/themes/SideNavigation.css.js";
+import SideNavigationCss from "./generated/themes/SideNavigation.css";
 import SideNavigationPopoverCss from "./generated/themes/SideNavigationPopover.css.js";
 
-/**
- * @public
- */
-const metadata = {
-	tag: "ui5-side-navigation",
-	managedSlots: true,
-	fastNavigation: true,
-	properties: /** @lends sap.ui.webc.fiori.SideNavigation.prototype */ {
-		/**
-		 * Defines whether the <code>ui5-side-navigation</code> is expanded or collapsed.
-		 *
-		 * @public
-		 * @type {boolean}
-		 * @defaultvalue false
-		 */
-		collapsed: {
-			type: Boolean,
-		},
-
-		/**
-		 * @private
-		 */
-		_popoverContent: {
-			type: Object,
-		},
-	},
-	slots: /** @lends sap.ui.webc.fiori.SideNavigation.prototype */ {
-		/**
-		 * Defines the main items of the <code>ui5-side-navigation</code>. Use the <code>ui5-side-navigation-item</code> component
-		 * for the top-level items, and the <code>ui5-side-navigation-sub-item</code> component for second-level items, nested
-		 * inside the items.
-		 *
-		 * @public
-		 * @type {sap.ui.webc.fiori.ISideNavigationItem[]}
-		 * @slot items
-		 */
-		"default": {
-			propertyName: "items",
-			invalidateOnChildChange: true,
-			type: HTMLElement,
-		},
-
-		/**
-		 * Defines the header of the <code>ui5-side-navigation</code>.
-		 *
-		 * <br><br>
-		 * <b>Note:</b> The header is displayed when the component is expanded - the property <code>collapsed</code> is false;
-		 *
-		 * @public
-		 * @type {HTMLElement[]}
-		 * @since 1.0.0-rc.11
-		 * @slot
-		 */
-		header: {
-			type: HTMLElement,
-		},
-
-		/**
-		 * Defines the fixed items at the bottom of the <code>ui5-side-navigation</code>. Use the <code>ui5-side-navigation-item</code> component
-		 * for the fixed items, and optionally the <code>ui5-side-navigation-sub-item</code> component to provide second-level items inside them.
-		 *
-		 * <b>Note:</b> In order to achieve the best user experience, it is recommended that you keep the fixed items "flat" (do not pass sub-items)
-		 *
-		 * @public
-		 * @type {sap.ui.webc.fiori.ISideNavigationItem[]}
-		 * @slot
-		 */
-		fixedItems: {
-			type: HTMLElement,
-			invalidateOnChildChange: true,
-		},
-	},
-	events: /** @lends sap.ui.webc.fiori.SideNavigation.prototype */ {
-		/**
-		 * Fired when the selection has changed via user interaction
-		 *
-		 * @event sap.ui.webc.fiori.SideNavigation#selection-change
-		 * @param {HTMLElement} item the clicked item.
-		 * @allowPreventDefault
-		 * @public
-		 */
-		"selection-change": {
-			detail: {
-				item: {
-					type: HTMLElement,
-				},
-			},
-		},
-	},
-};
+interface ISideNavigationItem extends UI5Element {
+	items: Array<ISideNavigationItem>;
```

Comment:
from what I understand, the SideNavigation should only contain SideNavigationItem instances, while the SideNavigationItem should only contain SideNavigationSubItem instances.

If this interface suggests interoperability between both SideNavigationItem and SideNavigationSubItem instances, then try to think of a different name for it

Reply:
Done.